config/sync/search_api.index.default_index.yml

```diff
@@ -2,35 +2,29 @@ uuid: b11493ed-029b-4616-a795-2a64321f5b9c
 langcode: en
 status: true
 dependencies:
-  module:
-    - node
-    - search_api
   config:
-    - field.storage.node.field_description
+    - field.storage.node.field_type_of_food
     - search_api.server.default_server
     - core.entity_view_mode.node.search_index
+  module:
+    - search_api
+    - node
+    - taxonomy
 _core:
   default_config_hash: GLmLegZGHtMF6annRqgjizJAoiAip-IT-XOsDMCZfog
 id: default_index
 name: 'Default content index'
 description: 'Default content index created by the Database Search Defaults module'
 read_only: false
 field_settings:
-  field_description:
```

Comment:
I think you might have removed data to be indexed that you shouldn't have. Seems like only the facets work now, the search bar doesn't:

![Screenshot_20200416_124219](https://user-images.githubusercontent.com/13573052/79447318-fac6f980-7fdf-11ea-86d8-63a9b2052b3b.png)
![Screenshot_20200416_124227](https://user-images.githubusercontent.com/13573052/79447314-fa2e6300-7fdf-11ea-8c1b-73e89e1ce77e.png)

Reply:
Fixed!